### DISCUSSION TITLE
Small Object Detection: Shrimp Seed Detection

### DISCUSSION THREAD

**Discussion by nandotmbn** [AlexeyAB/darknet#8535] [Q&A]
I have dataset for Shrimp Seed Detection, an image has many small object, are these working well or not?
Or maybe i have to change my dataset to image with few large object ?
Small Object
![Many Small Object](https://user-images.githubusercontent.com/66873366/170929953-41d3a4fd-2e48-4e04-b246-05456045c93b.jpg )

**Comment by stephanecharette** [comment]
See https://www.youtube.com/watch?v=861LvUXvJmA and https://www.ccoderun.ca/darkhelp/api/Tiling.html
